### ISSUE TITLE
Uso del -filter

### ISSUE THREAD

**Issue by Speedfroggy** [ctt-gob-es/clienteafirma#158]
Hola de nuevo

Tampoco me queda muy claro el uso del parámetro -filter
Este un ejemplo de llamada echa desde CMD en Windows
`C:\Progra~1\Autofirma\Autofirma\AutoFirma sign -filter subject.contains:123 -certgui -i C:\Temp\not_is20206900735525.pdf -o C:\Temp\not_is20206900735525-SIGNED.pdf -format auto -store auto -config layer2Text="Firmado por $$SUBJECTCN$$ el día $$SIGNDATE=dd/MM/yyyy$$ Con certificado emitido por $$ISSUERCN$$"\nsignaturePositionOnPageLowerLeftX=100\nsignaturePositionOnPageLowerLeftY=100\nsignaturePositionOnPageUpperRightX=400\nsignaturePositionOnPageUpperRightY=200`

Lo que busco es que solo salgan los certificados en los que el usuario tenga 123 en el DNI.
Si lo lanzo, no parece hacer nada y muestra todos los certificados. He probado varias maneras (con :, con =, con *…) pero no lo consigo.

Gracias

**Comment by Speedfroggy** [comment]
Me contesto a mi mismo.
-certgui parece incompatible con el uso de -filter.

Si la llamada de arriba se quita -certgui funciona

**Comment by doncicuto** [comment]
Efectivamente, certgui y filter son excluyentes. Si se quiere que en el selector de certificados (-certgui) se muestren sólo por ejemplo certificados de la FNMT se podría añadir: `-config filters=issuer.contains:FNMT`

Los filtros que se pueden utilizar se pueden encontrar en el Manual de Integrador : [https://administracionelectronica.gob.es/ctt/resources/Soluciones/138/Descargas/Manual%20Integrador%201-7-0.pdf?idIniciativa=138&idElemento=18913](https://administracionelectronica.gob.es/ctt/resources/Soluciones/138/Descargas/Manual%20Integrador%201-7-0.pdf?idIniciativa=138&idElemento=18913)
